persist/fs/retriever.go

```diff
@@ -313,6 +315,13 @@ func (r *blockRetriever) fetchBatch(
 
 		req.onRetrieved(seg)
 	}
+
+	err = seekerMgr.Return(shard, blockStart, seeker)
+	if err != nil {
+		r.fsOpts.InstrumentOptions().Logger().Errorf(
+			"err returning seeker for shard: %d, and blockStart: %d, err: %s",
+			shard, blockStart.Unix(), err.Error())
```

Comment:
nit: Maybe use `WithFields(...)` here?  Having a lot of vars in the fmt string doesn't seem fantastic.

Reply:
yeah fair